src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -451,20 +450,49 @@ export class BucketDetailComponent implements OnInit {
   //Show Restore Form
   showRestoreObject(file){
     this.restoreDisplay = true;
+    if(this.bucketBackendType=='azure-blob'){
```

Comment:
Is it possible to use some constant file and read these from there. So no need to remove across
only change that constant file

Reply:
I think it should be possible. let me see what I can do.